gen_cert.py

```diff
@@ -2065,6 +2065,12 @@ def _generate_v4_certificate(
                             continue
                         draw_template_element(self, element, attributes, page)
 
+        # Render any badges
```

Comment:
hmm, either one would be useful...

if it were before we could do the CS101 `1   0   1   0` background thing

but if it's after we could use it to add watermarking...